application/config/config.go

```diff
@@ -1119,3 +1121,23 @@ func (c *Config) SetSnykOpenBrowserActionsEnabled(enable bool) {
 	defer c.m.Unlock()
 	c.isOpenBrowserActionEnabled = enable
 }
+
+func (c *Config) FolderConfig(path string) *types.FolderConfig {
+	var folderConfig *types.FolderConfig
+	var err error
+	folderConfig, err = gitconfig.GetOrCreateFolderConfig(path)
+	if err != nil {
+		folderConfig = &types.FolderConfig{}
+	}
+	c.m.RLock()
+	folderConfig.AdditionalParameters = c.folderAdditionalParameters[path]
```

Comment:
we assume here that we call SetAdditionalParameters before c.FolderConfig
might be a nitpick but should we check here if the key exists in the map firstfirst?